### ISSUE TITLE
Warning: Calling 'depends_on :ant' is deprecated!

### ISSUE THREAD

**Issue by lexaurin** [bramstein/homebrew-webfonttools#22]
There is some warning when installing on HighSierra 10.13.3, Homebrew 1.5.11

```
$ brew tap bramstein/webfonttools

==> Tapping bramstein/webfonttools
Cloning into '/usr/local/Homebrew/Library/Taps/bramstein/homebrew-webfonttools'...
remote: Counting objects: 11, done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 11 (delta 0), reused 5 (delta 0), pack-reused 0
Unpacking objects: 100% (11/11), done.
Warning: Calling 'depends_on :ant' is deprecated!
Use 'depends_on "ant"' instead.
/usr/local/Homebrew/Library/Taps/bramstein/homebrew-webfonttools/sfntly.rb:7:in `<class:Sfntly>'
Please report this to the bramstein/webfonttools tap!
```

**Comment by filoxo** [comment]
Looks like the warning is now an error in Homebrew 1.6.0 (git revision 5bd3; last commit 2018-04-11).

```
==> Tapping bramstein/webfonttools
Cloning into '/usr/local/Homebrew/Library/Taps/bramstein/homebrew-webfonttools'...
remote: Counting objects: 11, done.
remote: Compressing objects: 100% (11/11), done.
remote: Total 11 (delta 0), reused 5 (delta 0), pack-reused 0
Unpacking objects: 100% (11/11), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/bramstein/homebrew-webfonttools/sfntly.rb
Calling 'depends_on :ant' is disabled!
Use 'depends_on "ant"' instead.
/usr/local/Homebrew/Library/Taps/bramstein/homebrew-webfonttools/sfntly.rb:7:in `<class:Sfntly>'
Please report this to the bramstein/webfonttools tap!
Or, even better, submit a PR to fix it!
Error: Cannot tap bramstein/webfonttools: invalid syntax in tap!
```

**Comment by jackjennings** [comment]
There are now two pull requests open to fix this: #21 and #23. @bramstein ?

**Comment by bramstein** [comment]
This has been merged in now.
